manifests/s/SunshineStream/Sunshine/0.14.0/SunshineStream.Sunshine.locale.en-US.yaml

```
@@ -0,0 +1,32 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: SunshineStream.Sunshine
+PackageVersion: 0.14.0
+PackageLocale: en-US
+Publisher: SunshineStream
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author: 
+PackageName: Sunshine
```

```suggestion
PackageName: SunshineStream
```
This must match Control Panel.